src/start.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose')
+const server = require('./server.js')
+
+// connect to the db
+require('../db/connect.js')
+const db = mongoose.connection
+
+const port = process.env.PORT || 3000
+
+db.on('error', err => console.log('connection error: ' + err))
```

Comment:
I think it's better to use:
```js
db.on('error', console.error.bind(console, 'connection error:'));
```

Tbh, I don't really know why and have researched it before, but it's in the [docs for mongoose](http://mongoosejs.com/docs/).

https://www.tjvantoll.com/2015/12/29/console-error-bind/